dpdk/vr_dpdk_ethdev.c

```diff
@@ -162,6 +162,27 @@ vr_dpdk_ethdev_ready_queue_id_get(struct vr_interface *vif)
     return VR_DPDK_INVALID_QUEUE_ID;
 }
 
+/* Release ethdev RX queue */
+static void
+dpdk_ethdev_rx_queue_release(unsigned lcore_id, struct vr_interface *vif)
```

Comment:
In vr_dpdk_lcore_if_schedule() we assign as many RX queues as hardware lets us. In vr_dpdk_ethdev_rx_queue_init() we assign qp_release_op. Then in dpdk_lcore_rxtx_release() we call those releasing functions only for queues which have it assigned.